setup.py

```diff
@@ -1,20 +1,17 @@
 from setuptools import setup
 
-readme = open('README.rst').read()
 history = open('CHANGES.rst').read().replace('.. :changelog:', '')
 
 setup(
-    name='sqlalchemy-redshift',
+    name='gx-sqlalchemy-redshift',
     version='0.8.15.dev0',
-    description='Amazon Redshift Dialect for sqlalchemy',
-    long_description=readme + '\n\n' + history,
+    description='Forked Amazon Redshift Dialect for sqlalchemy',
+    long_description=history,
     long_description_content_type='text/x-rst',
-    author='Matt George',
-    author_email='mgeorge@gmail.com',
-    maintainer='Thomas Grainger',
-    maintainer_email='sqlalchemy-redshift@graingert.co.uk',
+    author='The Great Expectations Team',
+    author_email='team@greatexpectations.io',
     license="MIT",
-    url='https://github.com/sqlalchemy-redshift/sqlalchemy-redshift',
+    url='https://github.com/great-expectations/sqlalchemy-redshift',
     packages=['sqlalchemy_redshift', 'redshift_sqlalchemy'],
     package_data={'sqlalchemy_redshift': ['redshift-ca-bundle.crt']},
     python_requires='>=3.4',
```

Comment:
We can change this to 3.9. That's the min version we support for gx core.

Reply:
Also the list of python versions below may need to be updated. Currently we support python 3.9-3.12.

Reply:
There's a list of different entrypoints. I've only tested with psycopg2 so maybe we should delete the rest? It may be fine but I haven't tested.